File: client/src/models/Project.js
export class Project{
    constructor(data){
        this.id = data.id 
        this.name = data.name
        this.img = data.img
        this.description = data.description
        this.creator = data.creator
        this.creatorId = data.creatorId
        this.createdAt = new Date(data.createdAt).toLocaleDateString()
        this.updatedAt = new Date(data.updatedAt).toLocaleDateString()
      }
}

File: server/Models/Project.cs
namespace mobSite.Models;
public class Project
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Name { get; set; }
    public string Img { get; set; }
    public string Description { get; set; }
    public string Category { get; set; }
    public string CreatorId { get; set; }
    public Account Creator { get; set; }
}